tests/unit/lms/resources/lti_launch_test.py

```diff
@@ -15,9 +16,10 @@
 class TestHGroup:
     @pytest.mark.usefixtures("has_course")
     def test_it(self, lti_launch, course_service):
-        course_service.upsert.return_value = mock.sentinel.course
+        course = factories.Course()
```

Comment:
Using a "real" object instead of a sentinel here so the `upsert.return_value.id` check works.